pom.xml

```diff
@@ -1402,11 +1402,6 @@
     <!-- Profile to run benchmarks when the `benchmark` property is set -->
     <profile>
       <id>jmh-benchmark</id>
-      <activation>
-        <property>
-          <name>benchmark</name>
```

Comment:
This might affect the buildPlugin pipeline

Reply:
good catch - filed https://github.com/jenkins-infra/pipeline-library/pull/124